recipe/session/session_test.go

```diff
@@ -184,6 +168,101 @@ func TestCookieBasedAuth(t *testing.T) {
 	})
 }
 
+func TestRefreshAPI(t *testing.T) {
+	BeforeEach()
+
+	unittesting.SetKeyValueInConfig("access_token_validity", "2")
```

Comment:
Moved the refresh API tests to a new function to avoid using a short access token validity for all the tests in the above function.